package.json

```diff
@@ -131,8 +131,7 @@
     "test": "vitest --mode development",
     "test:rust": "(cd rust && just test && just lint)",
     "test:snapshots": "cross-env TARGET=web NODE_ENV=development playwright test --config=playwright.config.ts --grep=@snapshot --trace=on",
-    "test:unit": "vitest run --mode development --exclude **/jest-component-unit-tests/*",
-    "test:unit:components": "jest -c jest-component-unit-tests/jest.config.ts --rootDir jest-component-unit-tests/",
```

Comment:
`jest` is blown away. `npm run test:unit` will cover the new component files. Vitest does all the heavy lifting!

Reply:
Awesome! Please delete `npm run test:unit:components` from the `Makefile` as well.